examples/atlas/rename-cluster.md

```diff
@@ -0,0 +1,20 @@
+# Renaming a Cluster
+
+Currently, you're not able to easily rename a cluster, 
+however you could deploy a new cluster with the same properties and only change the name.
```

Comment:
[q] does the new cluster contain the data stored in the old cluster? I think yes but I am not sure.